### ISSUE TITLE
Django REST framework integration

### ISSUE THREAD

**Issue by barseghyanartur** [barseghyanartur/django-fobi#38]
fobi.contrib.apps.drf_integration
========================
``Django REST framework`` integration sub-package for ``django-fobi``.

Do you have forms that would want to make available via REST API? Then this sub-package is for you.
You will have LIST/OPTIONS/POST actions on your existing ``django-fobi`` forms.

Planned to be released before 13th of May, 2017.

Most of the common ``django-fobi`` fields are planned to be supported.

- boolean
- checkbox_select_multiple
- date
- date_drop_down
- datetime
- decimal
- email
- file
- float
- hidden
- input (some sort of a copy of `text` plugin)
- integer
- ip_address
- null_boolean
- password (some sort of a copy of `text` plugin)
- radio
- range_select
- regex
- select
- select_multiple
- select_multiple_with_max
- slider (just a copy of range_select, for compatibility with main package)
- slug
- text
- textarea (some sort of a copy of `text` plugin)
- time
- url

The more complex plugins, such as ``select_model_object`` plugin (and related) are planned to be supported in upcoming versions. Same goes for presentational plugins, such as ``content_image`` plugin (and related).

**Comment by Psykar** [comment]
Are form submission endpoints planned to be under DRF? Or are you envisioning setting up viewsets / routers in DRF before you can submit to the forms?

(I'm amazed that I just discover this package, to find you're adding the feature I really need apparently 9 days from now. No pressure on that timeline 😉  )

**Comment by barseghyanartur** [comment]
This is how it's gonna integrate.

```python
# Conditionally including django-rest-framework integration app
if 'fobi.contrib.apps.drf_integration' in settings.INSTALLED_APPS:
    from fobi.contrib.apps.drf_integration.urls import fobi_router
    urlpatterns += [
        url(r'^api/', include(fobi_router.urls))
    ]
```

**Comment by barseghyanartur** [comment]
[Sneak preview](https://github.com/barseghyanartur/django-fobi/tree/django-rest-framework-integration) of yet unfinished work.

**Comment by barseghyanartur** [comment]
Implemented (and released) in version [0.11](https://pypi.python.org/pypi/django-fobi/0.11).
